index.html

```diff
@@ -1083,6 +1090,82 @@ <h3>Multikey</h3>
 
           </section>
 
+          <section>
+            <h3>JsonWebKey</h3>
+            <p>
+The JSON Web Key data model is a specific type of <a>verification method</a>
+that utilizes the JWK specification [[RFC7517]] to encode key types into a
+set of parameters. To encode a JSON Web Key, the <a>verification method</a>
+`type` MUST be set to `JsonWebKey` and the `publicKeyJwk` value MUST be a
+valid JWK value as described in [[RFC7517]]. An example of a JSON Web Key is
+provided below:
+            </p>
+
+            <pre class="example nohighlight"
+              title="JSON Web Key encoding of a Ed25519 public key">
+{
+  "@context": ["https://www.w3.org/ns/security/jwk/v1"],
+  "id": "did:example:123456789abcdefghi#key-1",
+  "type": "JsonWebKey",
+  "controller": "did:example:123456789abcdefghi",
+  "publicKeyJwk": {
+    "kty": "OKP",
+    "alg": "EdDSA"
+    "crv": "Ed25519",
+    "kid": "key-1",
+    "x": "_1EiHquO2aUx9JARSu0P8jdYT_OVneYxYOnOMAmUcFI",
+  }
+}
+            </pre>
+
+            <p>
+In the example above, the `publicKeyJwk` value contains the JSON Web Key.
+The `kty` property encodes the key type of "OKP", which means
+"Octet string key pairs". The `alg` property identifies the algorithm intended
+for use with the public key. The `crv` property identifies the particular
+curve type of the public key. The `kid` property specifies how the public key
+might be referenced in software systems; if present, the value SHOULD match the
+`id` property of the encapsulating `JsonWebKey` object. Finally, the `x`
+property specifies the point on the Ed25519 curve that is associated with the
+public key.
+            </p>
+
+            <p>
+The `publicKeyJwk` property MUST NOT contain any property marked as
+"Private" in any registry contained in the JOSE Registries [[JOSE-REGISTRIES]].
+            </p>
+
+            <p>
+The JSON Web Key data model is also capable of encoding secret keys, sometimes
+referred to as <em>private keys</em>.
+            </p>
+
+            <pre class="example nohighlight"
+              title="JSON Web Key encoding of a Ed25519 secret key">
+{
+  "@context": ["https://www.w3.org/ns/security/jwk/v1"],
+  "id": "did:example:123456789abcdefghi#key-1",
+  "type": "JsonWebKey",
+  "controller": "did:example:123456789abcdefghi",
+  "secretKeyJwk": {
+    "kty": "OKP",
+    "alg": "EdDSA"
+    "crv": "Ed25519",
+    "kid": "key-1",
+    "d": "Q6JwjCUdThSnoxfXHSFt5C1nVFycY_ZpW7qVzK644_g",
+    "x": "_1EiHquO2aUx9JARSu0P8jdYT_OVneYxYOnOMAmUcFI",
+  }
+}
+            </pre>
+
+            <p>
+The example above is identical to the previous example, except that the
+information is stored in the `secretKeyJwk` property and its value encodes the
+private key value in the `d` property.
```

Comment:
this is definitely not true for all secret keys, and may yet be changing - why would we define any of this here rather than just point at rfc7517?

Reply:
See https://datatracker.ietf.org/doc/draft-ietf-cose-key-thumbprint/

for concrete examples...

https://mailarchive.ietf.org/arch/msg/cose/T4Od25YoPmgOuALuuIGf1m8N4wo/

Reply:
> This seems possibly not true for all secret keys.

The language you are pointing to is providing a description of a specific example. It is not claiming that the example is true for all secret keys.

> this is definitely not true for all secret keys, and may yet be changing - why would we define any of this here rather than just point at rfc7517?

Please re-read the /description of the example/. We are not defining anything here, merely explaining what an example is expressing.